packages/webapp/src/delegates/components/delegate-funds-available.tsx

```diff
@@ -0,0 +1,97 @@
+import { useState } from "react";
+import {
+    assetToString,
+    Button,
+    Heading,
+    onError,
+    sumAssetStrings,
+    Text,
+    useDistributionsForAccount,
+    useMemberByAccountName,
+    useUALAccount,
+} from "_app";
+
+import { withdrawDelegateAvailableFunds } from "../transactions";
+
+interface Props {
+    account: string;
+}
+
+export const DelegateFundsAvailable = ({ account }: Props) => {
+    const [ualAccount] = useUALAccount();
+    const { data: member } = useMemberByAccountName(account);
+    const { data: distributions } = useDistributionsForAccount(account);
+    const [isLoading, setIsLoading] = useState(false);
+
+    const availableFunds = distributions
+        ? sumAssetStrings(
+              distributions.map((distribution) => distribution.balance)
+          )
+        : undefined;
+
+    // omit component if it's not a current delegate and available funds are empty
+    if ((!member || !member.election_rank) && !availableFunds) {
+        return null;
+    }
```

Comment:
A member could be allocated funds shortly before a new election and not withdraw them in time. We need to allow any active member (even if they're no longer a delegate) to withdraw when they have funds available to them in one or more distribution accounts.

Reply:
Thats exactly why I check available funds as well... if the member has not an election rank we will show the widget because it has available funds. Does it make sense?

Reply:
Awesome! I missed that. Thanks!